.github/workflows/Build.yml

```diff
@@ -35,11 +35,40 @@ jobs:
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
       - name: Configure AWS credentials for pytest
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
What is that change: aws-actions/configure-aws-credentials@v1-node16 ?

Reply:
We need to add node-16 to it, https://github.com/aws-actions/configure-aws-credentials#notice-node12-deprecation-warning